CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - Catch unnecessary warning while importing xgboost in CivisML_parallel_training.ipynb (#121)
 
+### Fixed
+- Fixed bug where instantiating a ``ModelPipeline`` from a CivisML model built w/ the templated script breaks (#122).
```

Comment:
This could be a bit clearer. All CivisML models use custom scripts created from the public templates, and not all `from_existing` calls fail. Perhaps instead of "from a CivisML model built w/ the templated script", use "from an existing model with empty "PARAMS" and "CV_PARAMS" boxes"? It would be helpful to call out `ModelPipeline.from_existing` explicitly, since that's where the problem was.